report/src/main/webapp/app/components/winterEdition.directive.js

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
the folder `report/src/main/webapp/app/components` is the old one and not realy in sync with John Papa guide about folders for code. let's move this directive to some folder with the functional-like name.

we can create new folder below `.../app` called *themes*.